### DISCUSSION TITLE
Attribute error while forecasting multiple time series

### DISCUSSION THREAD

**Discussion by widovanheemstra** [Nixtla/statsforecast#271] [Q&A] [answered]
Hi,
When I try to forecast using exogenous variables and only using AutoARIMA, I get an error stating "AttributeError: 'tuple' object has no attribute 'forecast'"

What am I doing wrong here?

Model defined as:

```
autoARIMA = AutoARIMA(season_length=12)
model = StatsForecast(
    df=train_df,
    models=[(autoARIMA,12)],
    freq="M",
    n_jobs=-1
)

fc = model.forecast(
    62,
    X_df=test_df,
    level=(80, 95)
)
```

**Comment by AzulGarza** [comment, marked as answer]
Hey @widovanheemstra! 

The problem arises due to the definition of the `models` parameter. Instead of using a tuple, you must include `autoARIMA` in the list.

The code would be,

```
autoARIMA = AutoARIMA(season_length=12)
model = StatsForecast(
    df=train_df,
    models=[autoARIMA],
    freq="M",
    n_jobs=-1
)

fc = model.forecast(
    62,
    X_df=test_df,
    level=(80, 95)
)
```

Tuples were deprecated as of v1.0.0 of StatsForecast.

Please let me know if another problem arises. :)

**Reply by widovanheemstra** [reply]
Thanks @FedericoGarza!
